osvr/RenderKit/osvr_display_configuration.h

```diff
@@ -69,6 +70,14 @@ class OSVRDisplayConfiguration {
     void OSVR_RENDERMANAGER_EXPORT
     parse(const std::string& display_description);
 
+    /// Produces a duplicate object that leaves out/defaults the parameters related to display output transformation
+    /// (e.g. rotation, swap eyes).
+    ///
+    /// @returns a unique_ptr because that's what factory functions that return reference objects are recommended to do
+    /// (they're convertable to shared_ptr, but not vice versa)
+    std::unique_ptr<OSVRDisplayConfiguration>
+    OSVR_RENDERMANAGER_EXPORT duplicateWithoutOutputTransforms() const;
```

Comment:
Where is this new method being used, and what does it have to do with the sensics display server changes? Something called from one of the NDA modules?